frontend/yarn.lock

```diff
@@ -12606,13 +12606,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"shared@npm:@opetushallitus/kieli-ja-kaantajatutkinnot.shared@1.9.21":
-  version: 1.9.21
-  resolution: "@opetushallitus/kieli-ja-kaantajatutkinnot.shared@npm:1.9.21::__archiveUrl=https%3A%2F%2Fnpm.pkg.github.com%2Fdownload%2F%40Opetushallitus%2Fkieli-ja-kaantajatutkinnot.shared%2F1.9.21%2F0c0b65a205f9bb1d94bcf9c69cd138c3c65462de"
-  checksum: ece19efc53162290166b93bbeb3e441df1abc2e2cad71790dec5d27ec20a7e4a61d89100c00b18cf47e21986698056525a0256a1395c738a34d41fd495399e21
-  languageName: node
-  linkType: hard
-
```

Comment:
Nyt kun katson niin tämä on vähän yllättävä, onkohan yarnin kansssa mennyt joku pieleen?

Reply:
Ei kai pitäis olla mitään väärin tässä. tuo 1.9.21 versio ei enää ole riippuvuutena niin se poistuu yarn.lockista.